Angular/src/app/api.service.ts

```diff
@@ -512,6 +533,10 @@ export class ApiService {
     return this.http.post<any>(this.NeuralURL + `scan_lines`, lines, { observe: 'body', responseType: 'json' });
   }
 
```

Comment:
Waar is dit precies voor?

Reply:
communicatie met het [Latin Scansion](https://github.com/Ycreak/Latin_scansion_with_neural_networks) project :)

Reply:
Ah. Pls add comment

Reply:
No.